app/views/properties/_warning_label.html.erb

```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body-s hackney-property-warning-label hackney-property-warning-label-<%= colour %>">
```

Comment:
I can't find where this partial is used in the code

Reply:
This was a mistake. Removing.

Reply:
Done. Also removed another stray template.

Reply:
Hahah yeah was just about to comment on that one